api/proxy/README.md

```diff
@@ -283,7 +283,40 @@ $ set -a; source ./.env; set +a; ./bin/eigenda-proxy
 
 ### Features and Configuration Options (flags/env vars)
 
-Below is a list of the main high-level features offered for configuring the eigenda-proxy. These features are controlled via flags and/or env vars. To view the extensive list of available flags/env-vars to configure a given version of eigenda-proxy, run `eigenda-proxy --help`.
+Below is a list of the main high-level features offered for configuring the eigenda-proxy. These features are
+controlled via flags and/or env vars. To view the extensive list of available flags/env-vars to configure a given
+version of eigenda-proxy, run `eigenda-proxy --help`.
+
+#### Payment Mode Configuration
+
+When using EigenDA V2, the payment system can be configured using the `--eigenda.v2.client-ledger-mode` flag (or the
+`EIGENDA_PROXY_EIGENDA_V2_CLIENT_LEDGER_MODE` environment variable). This flag determines which payment mechanisms are
+active for blob dispersals. For detailed information about the payment system, see the
+[payment system documentation](../../docs/spec/src/protocol/payments/payment_system.md).
+
+**Available Payment Modes:**
+
+1. **`legacy` (default)** - Uses the legacy bin-based payment system that handles both reservation and on-demand
+   payments. This mode is in the process of being deprecated and will be removed in a future release.
+   
+> **IMPORTANT**: All clients should continue using this mode until the new payment system has officially shipped. The
+> other payment modes are documented below for awareness, but they aren't ready to be used in production environments.
```

Comment:
shouldn't we say that they aren't supported by our currently deployed dispersers?

"not ready to be used in prod environments" still has the impliction that "if you are risk tolerant, you might want to use this beta feature"

Reply:
> "if you are risk tolerant, you might want to use this beta feature"

Well, I think this is pretty much correct. But I agree we should more explicitly discourage usage of the new payments for now

3a5ad0f67911da7a95b57496d09c473504863717

Reply:
Correct, it's possible to use the new configuration on clients, while the disperser still uses the old logic. 

This is actually being tested in the payments inabox tests